resources/lib/pigskin.py

```diff
@@ -363,11 +364,15 @@ def get_publishpoint_streams(self, video_id, stream_type=None, game_type=None, u
             if m3u8_obj.is_variant:  # if this m3u8 contains links to other m3u8s
                 for playlist in m3u8_obj.playlists:
                     bitrate = int(playlist.stream_info.bandwidth)
+                    resolution = (str(playlist.stream_info.resolution)).replace(', ','x')
                     streams[bitrate] = m3u8_url[:m3u8_url.rfind('/manifest') + 1] + playlist.uri + '?' + m3u8_url.split('?')[1] + '|' + urllib.urlencode(m3u8_header)
+                    streamResolutionInformation[bitrate] = resolution
```

Comment:
It's a style guide on how to write good and readable Python code. It's nothing wrong with using camel case variables: it's just that we should be consistent and not mix naming styles.